### ISSUE TITLE
Remove whitespace / add minheight

### ISSUE THREAD

**Issue by Aichnerc** [ricardo-ch/react-easy-crop#218]
**Is your feature request related to a problem? Please describe.**
|Current|Expectations|
|-|-|
|Whitespace under the image|Image fills the container| 
| ![image](https://user-images.githubusercontent.com/21159914/110970589-2ff7db00-835a-11eb-99fd-0e3feece4b91.png) | ![image](https://user-images.githubusercontent.com/21159914/110970625-371ee900-835a-11eb-8cdd-ce46429eedc0.png) |

**Describe the solution you'd like**
It would be nice to have a parameter to always enable fit to container. I use `cropSize`, which disables responsive image adaption. It would be nice to have functionality to still adapt a minimum image size.

**Comment by ValentinH** [comment]
This is the issue when using `cropSize`, you are losing all benefits from this library. To be honest, I actually regret having accepted to add this option as it goes against the initial philosophy of this project.

I'm wondering if you could meet your initial objective by using `aspect=1`, setting the initial zoom to a value higher than 1 and add padding to the parent container.

**Comment by ValentinH** [comment]
Closing as there was no answer for a while. Comment to re-open 😉

**Comment by Aichnerc** [comment]
So for anyone having the same goals, I fixed it as follows:

First, I initialized a new image instance.
`let img = new Image();`

Within the `img.onload` I then got the height and width of the image.
```
const height = img.height;
const width = img.width;
```

I then used the aspect ratio as zoom value, checking if I don't divide the smaller side by the bigger side.
`const zoomValue = height > width ? height / width : width / height;`

(optional) I then rounded the zoom value up to the next decimal point and used the value as `minZoom` at the `<Cropper />`.
`minZoom={this.state.minZoom ? this.state.minZoom : 1}`
